website/blog/composite-transport/index.mdx

```diff
@@ -0,0 +1,117 @@
+* Different processing: The Composite Transport allows you to send events to different destinations for different types of processing, such as logging and analytics.
+
+# How to Use the Composite Transport
+
+To use the Composite Transport, you can configure it in two formats: a list of transport configurations or a map of transport configurations. The list format is useful when you have a fixed set of transports, while the map format is useful when you need to configure transports dynamically.
+
+Here are some examples of how to use the Composite Transport in Python and Java:
+
+<Tabs groupId="examples">
+<TabItem value="python" label="Python Example">
+
+```python
+from openlineage.client import OpenLineageClient
+from openlineage.client.transport.composite import CompositeTransport, CompositeConfig
+
+config = CompositeConfig.from_dict({
+    "type": "composite",
+    "transports": [
+        {
+            "type": "kafka",
+            "config": {"bootstrap.servers": "localhost:9092"},
+            "topic": "random-topic",
+            "messageKey": "key",
+            "flush": False,
+        },
+        {"type": "console"},
+    ],
+})
+
+client = OpenLineageClient(transport=CompositeTransport(config))
+```
+</TabItem>
+
+<TabItem value="java" label="Java Example">
+
+```java
+import java.util.Arrays;
+import io.openlineage.client.OpenLineageClient;
+import io.openlineage.client.transports.CompositeConfig;
+import io.openlineage.client.transports.HttpConfig;
+import io.openlineage.client.transports.HttpTransport;
+import io.openlineage.client.transports.KafkaConfig;
+import io.openlineage.client.transports.KafkaTransport;
+
+HttpConfig httpConfig = new HttpConfig();
+httpConfig.setUrl("http://example.com/api");
+KafkaConfig kafkaConfig = new KafkaConfig();
+KafkaConfig.setTopicName("openlineage.events");
+KafkaConfig.setLocalServerId("some-value");
+
+CompositeConfig compositeConfig = new CompositeConfig(Arrays.asList(
+  new HttpTransport(httpConfig),
+  new KafkaTransport(kafkaConfig)
+), true);
+
+OpenLineageClient client = OpenLineageClient.builder()
+  .transport(
+    new CompositeTransport(compositeConfig))
+  .build();
+```
+</TabItem>
+
+<TabItem value="yaml-map" label="Yaml Config">
+
+```yaml
+transport:
+  type: composite
+  transports:
+    first:
+      type: http
+      url: http://example.com
+      auth:
+        type: api_key
+        apiKey: random_token
+      compression: gzip
+    second:
+      type: http
+      url: http://localhost:5050
+      endpoint: api/v1/lineage
+    debug:
+      type: console
+```
+</TabItem>
+
+</Tabs>
+
+# Conclusion
+
+The Composite Transport is a powerful feature in OpenLineage that allows you to send events to multiple destinations sequentially.
+With its flexibility, convenience, and redundancy, it's an essential tool for anyone working with data lineage.
```

Comment:
`With its flexibility, convenience, and redundancy, it's an essential tool for anyone working with data lineage.` -> What information is this sentence containing?